src/specmap/lib/all-of.js

```diff
@@ -75,6 +75,12 @@ export default {
       return undefined;
     });
 
+    // If there was an example in the original definition,
+    // keep it instead of merging with examples from subschemas
+    if (originalDefinitionObj.example) {
+      // Delete subschemas examples
+      patches.push(specmap.remove([].concat(parent, 'example')));
```

Comment:
What would you say if we simplify this statement using array literal instead?
```suggestion
      patches.push(specmap.remove([parent, 'example']));
```

I know there is this: https://github.com/swagger-api/swagger-js/blob/master/src/specmap/lib/all-of.js#L84 but it's unnecessary complex IMHO.

Reply:
If I understand correctly, the semantics differ. `concat` merges arrays, appending `'example'` to the `parent`. The literal creates the array nesting the `parent` inside.

For the `parent` equal to `['definitions', 'model']`, the array literal produces `[['definitions', 'model'], 'example']` instead of `['definitions', 'model', 'example']`.

The `remove` patch takes an array of strings as the `path` field. It may not handle the nested array correctly.

Reply:
You're absolutely right, I didn't realize that parent is an array.